Userland/Utilities/tail.cpp

```diff
@@ -62,7 +62,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (!follow)
         TRY(Core::System::pledge("stdio"));
 
-    auto file_is_seekable = !f->tell().is_error();
+    auto file_is_seekable = !f->seek(0, SeekMode::SetPosition).is_error();
```

Comment:
It's not really clear from your commit message what this change is fixing. You shouldn't need to seek to the beginning of a file to check whether it is seekable or not.

If you're trying to make `find_seek_pos()` work as expected when `start_from_end` is false, then I would suggest putting this seek call in the appropriate place in `find_seek_pos()`.

Reply:
It fixes exactly what it says it does. Some change in the past makes `f->tell()` not return an error if a file is not seekable. So in current master `echo "foo" | tail` crashes.

Reply:
Sorry, what I mean is, it isn't clear from the commit message *why* this change needed to be made.

From the [contribution guidelines](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md):

> [Don't] use weasel-words like "refactor" or "fix" to avoid explaining what's being changed.

So something like this would be better in this case:

> tail: Use `File::seek()` determine file seekability

---

> Some change in the past makes f->tell() not return an error if a file is not seekable.

Including something like this in the commit message body would be helpful here.